public/fonts/icomoon.svg

```diff
@@ -4,29 +4,30 @@
 <metadata>Generated by IcoMoon</metadata>
 <defs>
 <font id="icomoon" horiz-adv-x="1024">
-<font-face units-per-em="1024" ascent="819.2" descent="-204.8" />
+<font-face units-per-em="1024" ascent="819.8005865102639" descent="-204.19941348973606" />
```

Comment:
Instead of using icomoon's font file directly, I've gotten into the habit of copy-pasting just the glyph element into the existing SVG. Then upload that file to generate the TTF and WOFF files. It's a little roundabout, but it prevents drifting on these font metric attributes. 

So the operation is this.

1. Add your icon to an empty set and generate + download the fonts
2. Open the SVG and copy the glyph element. Paste into this file: *public/fonts/icomoon.svg*
3. Upload this file and select all the icons. Make sure to deselect the single icon set from 1.
4. Generate, download, move just the TTF and WOFF files to *public/fonts/*

Reply:
this is done